benchmarks/IO.py

```diff
@@ -180,6 +194,8 @@ def create_parser():
         raise ValueError("Dtype must be {}, not {}".format("/".join(TYPES), args.dtype))
     ak.verbose = False
     ak.connect(args.hostname, args.port)
+    comp_str = args.compressed
+    comp_types = None if comp_str == "" else comp_str.lower().split(",")
```

Comment:
Going off my previous comment, I think we want this to default to all compression types now.

Reply:
Changed this to default to `COMPRESSIONS`